e2e/tcproute_test.go

```diff
@@ -330,16 +338,20 @@ func TestRgraphTCPRouteAddBackends(t *testing.T) {
 		{Type: exec.ActionTypeCreate, Name: actionName(exec.ActionTypeCreate, tcprID)},
 		{Type: exec.ActionTypeCreate, Name: actionName(exec.ActionTypeCreate, bsID)},
 		{Type: exec.ActionTypeCreate, Name: actionName(exec.ActionTypeCreate, hcID)},
-		{Type: exec.ActionTypeMeta, Name: eventName(negID)},
+		{Type: exec.ActionTypeCreate, Name: actionName(exec.ActionTypeCreate, negID)},
```

Comment:
thisis changing the behavior of the test, isn't it?

Reply:
yes, previously there were hardcoded references to existing NEGs. Now they are created as needed for the duration of the test

Reply:
that's a good point, you don't have to change the behaviour of the test. Just create the NEG with raw API call and leave the node as it was (External).